assets/tsconfig.json

```diff
@@ -0,0 +1,64 @@
+        "lib": [
+            "dom",
+            "es2017"
+        ], /* Specify library files to be included in the compilation. */
+        "allowJs": true, /* Allow javascript files to be compiled. */
+        // "checkJs": true,                    /* Report errors in .js files. */
+        "jsx": "react", /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+        // "declaration": true,                /* Generates corresponding '.d.ts' file. */
+        // "declarationMap": true,             /* Generates a sourcemap for each corresponding '.d.ts' file. */
+        // "sourceMap": true,                  /* Generates corresponding '.map' file. */
+        // "outFile": "./",                    /* Concatenate and emit output to single file. */
+        // "outDir": "./",                     /* Redirect output structure to the directory. */
+        // "rootDir": "./",                    /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+        // "composite": true,                  /* Enable project compilation */
+        // "removeComments": true,             /* Do not emit comments to output. */
+        // "noEmit": true,                     /* Do not emit outputs. */
+        // "importHelpers": true,              /* Import emit helpers from 'tslib'. */
+        // "downlevelIteration": true,         /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
+        // "isolatedModules": true,            /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
+        /* Strict Type-Checking Options */
+        // "strict": true, /* Enable all strict type-checking options. */
+        // "noImplicitAny": true, /* Raise error on expressions and declarations with an implied 'any' type. */
+        "strictNullChecks": true, /* Enable strict null checks. */
+        // "strictFunctionTypes": true, /* Enable strict checking of function types. */
+        // "strictBindCallApply": true, /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
+        // "strictPropertyInitialization": true, /* Enable strict checking of property initialization in classes. */
+        // "noImplicitThis": true, /* Raise error on 'this' expressions with an implied 'any' type. */
+        // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
+        // /* Additional Checks */
+        // "noUnusedLocals": true, /* Report errors on unused locals. */
+        // "noUnusedParameters": true, /* Report errors on unused parameters. */
+        // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
+        // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
```

Comment:
Should we enable any of these Additional Checks or are they already covered by the linting?

Reply:
hmmm. i'll give it a little more thought. i just copied this over from 🏹